### ISSUE TITLE
Build fail for uTox 0.17.1 and GCC 9.2.1

### ISSUE THREAD

**Issue by EGD** [uTox/uTox#1421]
### Overview

1. uTox version: 0.17.1
2. Operating system: openSUSE Factory
3. The C compiler identification is GNU 9.2.1

### Issue

```
[   34s] cd /home/abuild/rpmbuild/BUILD/uTox/build/src/av && /usr/bin/cc  -I/usr/include/AL -isystem /home/abuild/rpmbuild/BUILD/uTox/third-party/stb -isystem /home/abuild/rpmbuild/BUILD/uTox/third-party/minini/dev -isystem /home/abuild/rpmbuild/BUILD/uTox/third-party/qrcodegen/c  -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG  -Wall -Wextra -Wpointer-arith -Wimplicit-fallthrough=5 -Werror=implicit-function-declaration -Wformat=0 -Wno-misleading-indentation -fno-strict-aliasing -fPIC -flto -DAUDIO_FILTERING=1 -O2 -g -DNDEBUG   -std=gnu11 -o CMakeFiles/utoxAV.dir/audio.c.o   -c /home/abuild/rpmbuild/BUILD/uTox/src/av/audio.c
[   34s] In file included from /home/abuild/rpmbuild/BUILD/uTox/src/av/audio.c:33:
[   34s] /usr/include/AL/alc.h:34:26: error: conflicting types for 'ALCdevice'
[   34s]    34 | typedef struct ALCdevice ALCdevice;
[   34s]       |                          ^~~~~~~~~
```

**Comment by avoidr** [comment]
This has been fixed in b608d584d9b6aeae916dec48c2cadf2e631f91b8. Try latest `develop` branch?

**Comment by EGD** [comment]
Thanks, this works.
